components/MainTiles/MainTiles.tsx

```diff
@@ -34,6 +35,7 @@ export const MainTiles = memo<MainTilesProps>((props) => {
 
   return (
     <section className={styles.section}>
+      <AlogliaSearch />
```

Comment:
Włóżmy to w diva o stałej wysokości i załadujmy ten komponent dynamicznie z ssr:false, żeby na dzień dobry nie ładować całego algoliasearch